We never generate fragments in /run for modutils to utilize, so disable that
logic entirely.  This avoids random exploits where tools can write to the /run
path and then trigger module loading via other means (which runs as root).

https://crbug.com/780039

diff --git a/libkmod/libkmod.c b/libkmod/libkmod.c
index 7c2b889..7cb959f 100644
--- a/libkmod/libkmod.c
+++ b/libkmod/libkmod.c
@@ -63,7 +63,6 @@ static struct _index_files {
 
 static const char *default_config_paths[] = {
 	SYSCONFDIR "/modprobe.d",
-	"/run/modprobe.d",
 	"/usr/local/lib/modprobe.d",
 	"/lib/modprobe.d",
 	NULL
diff --git a/tools/depmod.c b/tools/depmod.c
index 364b7d4..cc196de 100644
--- a/tools/depmod.c
+++ b/tools/depmod.c
@@ -52,7 +52,6 @@ static const char CFG_BUILTIN_KEY[] = "built-in";
 static const char CFG_EXTERNAL_KEY[] = "external";
 static const char *default_cfg_paths[] = {
 	SYSCONFDIR "/depmod.d",
-	"/run/depmod.d",
 	"/usr/local/lib/depmod.d",
 	"/lib/depmod.d",
 	NULL

